### DISCUSSION TITLE
Image Size

### DISCUSSION THREAD

**Discussion by yimuu** [Project-MONAI/MONAI#3925] [Q&A]
For a 3D image, the size is (D, H W) or  (W, H, D) ?

**Comment by Nic-Ma** [comment]
Hi @yimuu ,

I feel it's not very clear to define which dim is H and W in medical DL topics, @ericspod could you please help share more details?

Thanks in advance.

**Comment by ericspod** [comment]
I'm afraid the ordering varies, Nifti is XYZ which is typically interpreted as WHD, and Dicom is WH typically though specialized spatial formats may be WHD or CWHD. In Pytorch conv3d assumes ordering of the tensor is (N,Cin​,D,H,W) so the spatial dimensions there are DHW.
